test/models/config_test.rb

```diff
@@ -5,20 +5,34 @@ class ConfigTest < ActiveSupport::TestCase
 
   describe 'callbacks' do
     it 'should clean URLs before save' do
-      Config::CONFIG_URLS.each do |url_field|
-        c = Config.find_or_create_by(config_key: url_field)
-        
-        # confirm that scheme-less URLs are given one
-        c.config_value = { options: ["www.needs-scheme.net"] }
-        c.save
-        assert_equal "https://www.needs-scheme.net", c.options.last
-
-        # doesn't touch empty config
-        c.config_value = { options: [] }
-        c.save
-        assert_nil c.options.last
+      Config::CLEAN_PRE_VALIDATION
+        .select{ |field, cleaner| cleaner == :clean_url }
```

Comment:
Since you refactored out the clean step for URLs there are no cases where the cleaner is `:clean_url` anymore. I think this test is delete-able!